docs/en/sdk-manual/swift.md

```diff
@@ -22,13 +22,13 @@ We use **Swift Package Manager** to distribute Clickstream Swift SDK, open your
 
 ![](../images/sdk-manual/swift_add_package.png)
 
-Enter the Clickstream Library for Swift GitHub repo URL (`https://github.com/awslabs/clickstream-swift`) into the search bar, You'll see the Clickstream Library for Swift repository rules for which version of Clickstream you want Swift Package Manager to install. Choose **Up to Next Major Version**, then click **Add Package**, make the Clickstream product checked as default, and click **Add Package** again.
+Enter the Clickstream Library for Swift GitHub repo URL (`https://github.com/awslabs/clickstream-swift`) into the search bar, You'll see the Clickstream Library for Swift repository rules for which version of Clickstream you want Swift Package Manager to install. Choose **Up to Next Major Version**, then click **Add Package**, make the Clickstream product checked as default, and click **Add Package** again.
```

Comment:
Enter the Clickstream Library for Swift GitHub repository URL (`https://github.com/awslabs/clickstream-swift`) in the search bar, and then check the rules for the version of the solution that you want Swift Package Manager to install. Choose **Up to Next Major Version**, and choose **Add Package**, keep the Clickstream product checked, and choose **Add Package** again.

Reply:
Changed to:
1. Enter the Clickstream Library for Swift GitHub repository URL (`https://github.com/awslabs/clickstream-swift`) in the search bar.
2. Check the rules for the version of the solution that you want Swift Package Manager to install, it is recommended to choose **Up to Next Major Version**, then click **Add Package**.
3. Keep the Clickstream product checked as default.
4. Choose **Add Package** again.